.github/workflows/test.yaml

```diff
@@ -71,7 +74,25 @@ jobs:
           sudo add-apt-repository ppa:mozillateam/ppa
           printf 'Package: *\nPin: release o=LP-PPA-mozillateam\nPin-Priority: 1001' | sudo tee /etc/apt/preferences.d/mozilla-firefox
 
-          sudo apt install -y wget nodejs unzip firefox nginx
+          sudo apt install -y wget nodejs unzip firefox
+
+          # if nginx is already installed, remove it
+          sudo apt remove -y nginx nginx-common nginx-core
+
+          # install nginx from source so we can add the brotli module
+          git clone --recursive https://github.com/google/ngx_brotli.git
+          wget https://nginx.org/download/nginx-1.24.0.tar.gz
+          tar zxf nginx-1.24.0.tar.gz
+          cd ngx_brotli/deps/brotli
+          mkdir out && cd out
+          cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DCMAKE_C_FLAGS="-Ofast -m64 -march=native -mtune=native -flto -funroll-loops -ffunction-sections -fdata-sections -Wl,--gc-sections" -DCMAKE_CXX_FLAGS="-Ofast -m64 -march=native -mtune=native -flto -funroll-loops -ffunction-sections -fdata-sections -Wl,--gc-sections" -DCMAKE_INSTALL_PREFIX=./installed ..
+          cmake --build . --config Release --target brotlienc
+          cd ../../../..
+          export CURRENT_DIR=$(pwd)
+          cd nginx-1.24.0
+          ./configure --sbin-path=/usr/bin/nginx --conf-path=/usr/local/nginx/nginx.conf --pid-path=/usr/local/nginx/nginx.pid --with-http_ssl_module --with-stream --with-mail=dynamic --with-http_realip_module --with-compat --add-module=${CURRENT_DIR}/ngx_brotli
+          sudo make && sudo make install
+          sudo nginx
```

Comment:
It's nice to be able to test this, but maybe a bit much for an optional feature.

Reply:
I agree, was just a good way for me to get it to run on github and see if it works as expected. Will remove it!

Maybe we can keep that in skyportal though, and add the test there

Reply:
Or, if the simpler Debian instructions work, that'd be fine.

Reply:
Do you mean with libnginx-mod-http-brotli-filter? Well that was actually the issue for me, the lack of instruction. NGINX is supposed to be compile with the modules, so I didn't quite understand how installing this would work, nor what it did.

I can give it a go, but I found it more confusing than anything else.

Reply:
I think you just need to install this package and then you can use `load_module modules/ngx_http_brotli_filter_module.so;`.

Reply:
if that works that's great! But then it means I can't detect whether or not brotli is installed dynamically by running `nginx -V`, which I suspect this package doesn't edit. Could work too, I'll give it a shot.

Reply:
@stefanv a simple apt install can't actually find `libnginx-mod-http-brotli-filter`. Did you install it differently?